python_ta/patches/error_messages.py

```diff
@@ -4,18 +4,193 @@
+                      'defined have different names.',
+             'E0103': 'The %r keyword should only be used inside a loop.',
+             'E0104': 'The \'return\' keyword is used to end the execution of a function '
+                      'and return a result, so you should only use it inside of a function. '
+                      'Note that if no return statement is written, Python implicitly '
+                      'ends the execution of a function with return None',
+             'E0108': 'Duplicate parameter name %s in function definition. '
+             }
     },
     'pylint.checkers.refactoring': {
         'RefactoringChecker':
-             {'R1710': 'Since this function can return a non-None value, you should '
-                       'explicitly write "return None" whenever None is returned by '
-                       'this function. (Possibly including possibly at the end of the function body.)'
+            {'R1710': 'Since this function can return a non-None value, you should '
+                      'explicitly write "return None" whenever None is returned by '
+                      'this function. (Possibly including at the end of the function body.)'
              }
     },
     'pylint.checkers.design_analysis': {
-        'MisdesignChecker': {'R0913': 'Too many parameters (%s/%s)'}
+        'MisdesignChecker': {'R0912': 'The function has too many branches (%s out of a default limit '
+                                      'of %s). You should try to reduce the complexity of the function '
+                                      'by splitting it up.',
+                             'R0913': 'The function has too many parameters (%s out of a default limit '
+                                      'of %s). You should try to reduce the complexity of the function '
+                                      'by splitting up it, or combining related objects as a single one.',
+                             'R0914': 'The function has too many local variables (%s out of a default limit '
+                                      'of /%s).',
+                             'R0915': 'The function or method has too many statements (%s out of a default limit '
+                                      'of /%s). It should be split up into smaller functions or methods.',
+                             'R0916': 'The if statement contains too many boolean expressions '
+                                      '(%s out of a default limit of %s). You should see if you can reduce '
+                                      'the amount of expressions, or split it into branches.',
+                             }
+    },
+    'pylint.checkers.classes': {
+        'ClassChecker': {'E0203': 'You can\'t use class member %r prior to it\'s assignment '
+                                  'on line %s.',
+                         'E0211': 'Every class method requires at least one parameter, '
```

Comment:
class method -> instance method

parameter -> parameter, since the first parameter refers to the object on which this method is called. Python's convention is to name this parameter 'self'.